src/components/CreateTemplate/index.tsx

```diff
@@ -0,0 +1,210 @@
+  };
+
+  return (
+    <>
+      <Modal isOpen={isOpen} onClose={closeWindow} size="2xl" isCentered>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader role="modalTitle">Template sharing</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody padding="0.5rem 1.5rem">
+            <FormLabel>Court Preview:</FormLabel>
+            <Flex
+              fontSize="2rem"
+              fontWeight="500"
+              width="20rem"
+              height="11rem"
+              backgroundColor="orange"
+              justifyContent="center"
+              alignItems="center"
+              margin="1.8rem auto"
+            >
+              Court Image
+            </Flex>
+            <Flex gap="1.5rem" justifyContent="center" flexWrap="wrap">
+              <Badge margin="1rem" colorScheme="green">
+                {courtType}
+              </Badge>
+              <Badge margin="1rem">{selectedCourt.courtName}</Badge>
+            </Flex>
+            <Flex>
+              <FormControl
+                width="50%"
+                marginTop="1rem"
+                isRequired
+                isInvalid={inputError.courtNameFullErr}
+              >
+                <FormLabel marginBottom="1rem">Template Court Name:</FormLabel>
+                <Input
+                  role="courtNameInput"
+                  placeholder="Court name"
+                  width="15rem"
+                  onChange={checkNameLength}
+                  autoFocus
+                  ref={courtNameRef}
+                />
+              </FormControl>
+              <Box width="50%">
+                <Text margin="1rem 0px" fontSize="medium" fontWeight="500">
+                  Publisher:
+                </Text>
+                <Flex alignItems="center">
+                  <Icon as={FaUserCircle} fontSize="2.5rem" marginRight="1.8rem" />
+                  <Text
+                    fontSize="large"
+                    fontWeight="500"
+                    whiteSpace="nowrap"
+                    overflow="hidden"
+                    textOverflow="ellipsis"
+                  >
+                    {`${firstName} ${lastName}`}
+                  </Text>
+                </Flex>
+              </Box>
+            </Flex>
+            <ErrorMsg userInputError={inputError} inputErrorMsg={INPUT_ERROR_MSG} />
+            <FormLabel marginBottom="1rem">Description:</FormLabel>
+            <Textarea
+              height="9.5rem"
+              placeholder={`Description: maximum ${MAX_DESCRIPTION_LEN} words`}
+              onChange={handleTextAreaLenChange}
+              resize="none"
+              ref={descriptionRef}
+            />
+            <Text role="wordCount" color={textAreaLen < MAX_DESCRIPTION_LEN ? "black" : "crimson"}>
+              {inputError.descriptionOverLimit
+                ? INPUT_ERROR_MSG.descriptionLenErrMsg
+                : `${textAreaLen}/${MAX_DESCRIPTION_LEN} words`}
+            </Text>
```

Comment:
If you delete _FormControl_, _FormLabel_ can also be removed.

However, I think Robert means that if you use _FormControl_, you can use _FormErrorMessage_, you don't need to write _Text_ anymore, you can refer to the example he provided

Reply:
> If you delete _FormControl_, _FormLabel_ can also be removed.
> 
> However, I think Robert means that if you use _FormControl_, you can use _FormErrorMessage_, you don't need to write _Text_ anymore, you can refer to the example he provided

fixed